crates/sui-framework/packages/bridge/sources/bridge.move

```diff
@@ -37,6 +40,9 @@ module bridge::bridge {
         // Bridge treasury for mint/burn bridged tokens
         treasury: BridgeTreasury,
         bridge_records: LinkedTable<BridgeMessageKey, BridgeRecord>,
+        transfer_limits: VecMap<TypeName, VecMap<BridgeRoute, u64>>,
```

Comment:
instead of a per token limit, we want to use a global limit of all assets, with a fixed notional value. For example, say we fix USD value as follows:
BTC: 50000
ETH: 3000
USDC/USDT to 1

and global limit is 1M. In this case, when there are 200 BTC transferred, no any other assets can be transferred until clocks ticks to a certain point.

There are two things that can be changed with validator sigs:
1. the notional value of assets
2. the global limit